src/server.ts

```diff
@@ -1,10 +1,11 @@
-var express = require('express');
+import express from 'express'
```

Comment:
real, nunca entendi pq no node o pessoal usava require. tu sabe?

Reply:
Entao, acho que tem a ver com o padrao antigo. Antes do ES6 era assim que vc fazia para importar de outro modulo.
https://stackoverflow.com/questions/46677752/the-difference-between-requirex-and-import-x/46677972

Teoricamente *import* nao é nativo do commonJS. Mas nao me parece ser ruim.